### DISCUSSION TITLE
Is gps tracking supported?

### DISCUSSION THREAD

**Discussion by layonez** [strukturart/o.map#34] [Q&A] [answered]
There is no known to me good GPS tracker to track running cycling activity on kaios, but maybe that feature is already done by you or planned for the future?

**Comment by strukturart** [comment, marked as answer]
with o.map you can record your movement, the recording is saved as .geoJSON. however, this function is very basic, no altitude differences, heart rate etc.
In the appStore you can find an app that can do better: https://www.kaiostech.com/store/apps/?bundle_id=com.kaiostech.activity

It is certainly worth a try to use a dumbphone as a tracker, but I recommend you to use a device that was made for it, for example: a garmin smart watch.

I myself use o.map for climbing and cycling, for tracking I use a smartwatch.

**Reply by layonez** [reply]
Thank you @strukturart for a nice suggestion. I agree that that is not the intended use of my phone, I just actually experimenting in what areas I can do fine without a smartphone :)

**Comment by strukturart** [comment]
I've managed quite well with the dumbphone for the last 2 years, sometimes it takes a little more planning before you start but finally you have more time for other things.
